src/main/flight/pid.c

```diff
@@ -865,28 +878,13 @@ FAST_CODE float featheredPids(const pidProfile_t *pidProfile, int axis, float er
 {
     (void)(currentPidSetpoint);
 
-//Add EmuBoost to the code (non linear boost to errorRate)
-float errorMultiplier = (pidProfile->errorBoost * pidProfile->errorBoost / 1000000) * 0.003;
```

Comment:
This line was added to the code to allow for a nicer scaling of the emuboost values. It looks like nothing has been added to account for this. You think that we could keep this same scaling with the update?

Reply:
do the math.